.travis.yml

```diff
@@ -30,6 +30,7 @@ env:
 matrix:
     fast_finish: true
     allow_failures:
+      - env: TEST_SUITE=codecov
```

Comment:
I actually removed this on purpose. The test are allowed to fail (see gradle build file) but the build should fail if there is a problem with the code cov update.

Reply:
https://github.com/JabRef/jabref/pull/5618